references/detection/engine.py

```diff
@@ -44,8 +43,13 @@ def train_one_epoch(model, optimizer, data_loader, device, epoch, print_freq):
             sys.exit(1)
 
         optimizer.zero_grad()
-        losses.backward()
-        optimizer.step()
+        if scaler:
```

Comment:
Nit: Perhaps be explicit here as above:
```suggestion
        if scaler is not None:
```